src/index.css

```diff
@@ -2,3 +2,371 @@
   min-height: 100vh;
   height: 100vh;
 }
+
+
+head,
+body {
+  background-color: #ecf0f5;
+}
+.content-wrapper {
+  min-height: initial !important;
+}
+.bg-auth {
```

Comment:
@AtnNn Negative; ideally they should be scoped to a CSS module in the `auth` directory that the auth components then reference